<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn1
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn1:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn2
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn2:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn3
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn3:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn4
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn4:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn5
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn5:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn6
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn6:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn7
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn7:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn9
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn9:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn8
{background-image: url(:/btnClear.png); background-color:transparent;}
#btn8:hover 
{background-image: url(:/btnHover.png); background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="xSLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>592</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="oSLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>592</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="whoStepLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
